archunit/src/main/java/com/tngtech/archunit/library/dependencies/SlicesRuleDefinition.java

```diff
@@ -67,12 +67,28 @@ public GivenSlices matching(String packageIdentifier) {
             return new GivenSlicesInternal(Priority.MEDIUM, Slices.matching(packageIdentifier));
```

Comment:
We could reduce a tiny bit of code duplication if this now delegated to the new method:
```suggestion
            return matching(packageIdentifier, Priority.MEDIUM);
```